data/json/recipes/recipe_deconstruction_package.json

```diff
@@ -7,13 +7,13 @@
     "components": [
       [ [ "pur_tablets", 6 ] ],
       [ [ "gummy_vitamins", 1 ] ],
-      [ [ "coffee_raw", 1 ] ],
+      [ [ "instant_coffee", 1 ] ],
       [ [ "lemonade_powder", 1 ] ],
       [ [ "matches", 1 ] ],
       [ [ "gum", 2 ] ],
       [ [ "paper", 8 ] ],
       [ [ "spork", 1 ] ],
-      [ [ "bag_plastic", 1 ] ]
+      [ [ "beverage_bag", 1 ] ]
```

Comment:
Shouldn't disassembly recipe still create plastic bag (as it contains all the other items), as well as beverage bag? Unless the bag that contains the kit is actually the beverage bag, that is; I don't know if it's like that.

Reply:
<img width="240" alt="Screen Shot 2020-11-21 at 2 33 48 AM" src="https://user-images.githubusercontent.com/27952407/99831139-0136d880-2ba2-11eb-9c0c-de3f09d1a8cc.png">
<img width="240" alt="Screen Shot 2020-11-21 at 2 33 00 AM" src="https://user-images.githubusercontent.com/27952407/99831147-03993280-2ba2-11eb-8178-1fe4b1340640.png">

I've thought about it, but I feel like the plastic bag holding the accessory pack is too small and useless to count as a plastic bag item.